plugins/woocommerce-admin/client/tasks/fills/shipping/index.js

```diff
@@ -54,6 +53,8 @@ export class Shipping extends Component {
 		this.shippingSmartDefaultsEnabled =
 			window.wcAdminFeatures &&
 			window.wcAdminFeatures[ 'shipping-smart-defaults' ];
+
+		this.store_location_completed = false;
```

Comment:
Change this to `storeLocationCompleted` to follow the JS naming convention? 😅

Reply:
haha I'm sorry about that 😅 I was editing PHP before writing this line 😅

On it!

Reply:
@chihsuan Updated in https://github.com/woocommerce/woocommerce/pull/33661/commits/e9cfb2cc85ae7360e56b1123566d6eb38465fe2b

Reply:
@moon0326 Looks like this is changed back in the last commit. 😅  The others tested well and look good!